src/chrome/content/rules/MediaWorks.xml

```diff
@@ -35,7 +37,10 @@
 	<securecookie host="^(?:cdn|static)\.mediaworks\.co\.nz$" name=".+" />
 
 
-	<rule from="^http://(cdn|static)\.mediaworks\.co\.nz/"
-		to="https://$1.mediaworks.co.nz/" />
+	<rule from="^https?://cdn\.mediaworks\.co\.nz/"
```

Comment:
We try to avoid redirects from https, except it breaks something in the site (like wrong protocol relative redirects).